guiEditor/README-ES6.md

```diff
@@ -0,0 +1 @@
+Gui Editor
```

Comment:
would be great to extend this one a bit.

Reply:
Added :)